### DISCUSSION TITLE
There is no automatic prompt when entering the mysql keyword,  example ：left join .

### DISCUSSION THREAD

**Discussion by c384795430** [dbeaver/dbeaver#13614] [Q&A]
Does anyone know how to do it?

**Comment by LonwoLonwo** [comment]
Hello @c384795430 
Please show screens. Auto-complete in SQL Editor (are we talking about SQL Editor?) works and doesn't work in many situations. It depends on preferences and expression syntax.

**Comment by c384795430** [comment]
please help me.  
![setup](https://user-images.githubusercontent.com/88574668/130421948-072367af-8622-4b44-8aae-9dfcfee7012a.gif)

**Comment by LonwoLonwo** [comment]
Please add one more space after "join" and try to use Crtl+Space

![2021-08-23 12_31_26-](https://user-images.githubusercontent.com/45152336/130424808-b346ba43-4fbe-4a2a-8dfe-ec5077a959cc.png)

**Comment by c384795430** [comment]
Sorry, you may have misunderstood what I mean, what I mean is that there is no automatic completion when entering left join.

**Comment by LonwoLonwo** [comment]
Automatic completion for keywords? Like "left" or "join", or automatic completion for table name?

**Reply by c384795430** [reply]
Yes .

**Comment by LonwoLonwo** [comment]
You can create enhancement for this idea here: https://github.com/dbeaver/dbeaver/issues

**Comment by c384795430** [comment]
thank you .
